light/store/db/db.go

```diff
@@ -119,73 +105,43 @@ func (s *dbs) DeleteSignedHeaderAndValidatorSet(height int64) error {
 	return nil
 }
 
-// SignedHeader loads SignedHeader at the given height.
+// LightBlock retrieves the LightBlock at the given height.
 //
 // Safe for concurrent use by multiple goroutines.
-func (s *dbs) SignedHeader(height int64) (*types.SignedHeader, error) {
+func (s *dbs) LightBlock(height int64) (*types.LightBlock, error) {
 	if height <= 0 {
 		panic("negative or zero height")
 	}
 
-	bz, err := s.db.Get(s.shKey(height))
+	bz, err := s.db.Get(s.lbKey(height))
 	if err != nil {
 		panic(err)
 	}
 	if len(bz) == 0 {
-		return nil, store.ErrSignedHeaderNotFound
+		return nil, store.ErrLightBlockNotFound
 	}
 
-	var pbsh tmproto.SignedHeader
-	err = proto.Unmarshal(bz, &pbsh)
+	var lbpb tmproto.LightBlock
+	err = proto.Unmarshal(bz, &lbpb)
```

Comment:
```suggestion
	err = lbpb.Unmarshal(bz)
```
faster
![](https://media2.giphy.com/media/q6U4fbreC4Neo/giphy.gif)

Reply:
haha didn't know you could do that 👍